testing/btest/spicy/export-bitfield.zeek

```diff
@@ -0,0 +1,50 @@
+# @TEST-REQUIRES: have-spicy
+#
+# @TEST-EXEC: spicyz -do export.hlto export.spicy export.evt
+# @TEST-EXEC: zeek -Cr $TRACES/http/post.trace export.hlto %INPUT >>output
+# @TEST-EXEC: test '!' -e reporter.log
+# @TEST-EXEC: btest-diff output
+#
+# @TEST-DOC: Tests that named and anonymous bitfields are exported as expected.
+
+# @TEST-START-FILE export.spicy
+module foo;
+
+public type X = unit {
+    a: uint8;
+
+    bf: bitfield(8) {
+	 x1: 0..7;
+	 y1: 0..3;
+	 z1: 4..7;
+     };
+
+     : bitfield(8) {
+	 x2: 0..7;
+	 y2: 0..3;
+	 z2: 4..7;
+     };
+
+    b: uint8;
+};
+# @TEST-END-FILE
+
+# @TEST-START-FILE export.evt
+import foo;
+
+protocol analyzer FOO over TCP:
+    parse originator with foo::X,
+    port 80/tcp;
+
+export foo::X;
+
+on foo::X -> event foo::hello(self);
+# @TEST-END-FILE
+
+event zeek_init() {
+    print record_fields("foo::X");
+}
+
+event foo::hello(x: foo::X)
+	{ print x; }
```

Comment:
It would be great to also baseline the type of `foo::X::bf`, e.g., like this:

```suggestion
event foo::hello(x: foo::X)
	{
	print record_fields(x);
	print record_fields(x$bf);
	}
```